Remove cors decorator

qgerome opened the pull request:
The signal that I added prevent the frontend to load the static assets of the admin. By changing the implementation, I've fixed the static problem and kept the deactivation of the cors on the `run_pipeline` endpoint

cheikhgwane reviewed (approved):
Tested, lgtm

toch reviewed (commented):
> The signal that I added prevent the frontend to load the static assets of the admin.

Do you mean that when POSTing `run_pipeline`, the frontend was loading the static when it wasn't needed?

qgerome commented:
> > The signal that I added prevent the frontend to load the static assets of the admin.
> 
> Do you mean that when POSTing `run_pipeline`, the frontend was loading the static when it wasn't needed?

No. The `django-cors-headers` is supposed to used be able to work with the signal AND the settings variables but unfortunately it does not :/ I refactored my approach to just use their internal system to disable CORS for the calls to the pipelines to be able to run them from frontend apps (a request from a internal web app for a project)

qgerome commented on hexa/pipelines/views.py:
```diff
@@ -94,6 +95,7 @@ def credentials(request: HttpRequest) -> HttpResponse:
 
 @require_POST
 @csrf_exempt
+@disable_cors
```
Why do you mean "non public network"?

toch commented:
Thanks for the explanation. Is this endpoint can be called from the outside? What if we deploy each part separately, i.e. on different network/infra, and the communication go through the public network?

It's fine if this case is not handled securely, it means it is a current limitation that I would document.

qgerome commented:
> Thanks for the explanation. Is this endpoint can be called from the outside? What if we deploy each part separately, i.e. on different network/infra, and the communication go through the public network?
> 
> It's fine if this case is not handled securely, it means it is a current limitation that I would document.

I'm not sure to follow you. The goal of disabling the CORS for this endpoint is to allow browsers to call it from another domain of the Django app (or the ones defined in `CORS_ALLOWED_ORIGINS`. I don't understand the relation between this and the different network/infra/public network :/
